core-bundle/src/Image/ImageFactoryInterface.php

```diff
@@ -25,17 +25,13 @@ interface ImageFactoryInterface
      * @param string|ImageInterface              $path    The absolute path to the source image or an Image object
      * @param int|array|ResizeConfiguration|null $size    An image size ID, an array with width, height and resize mode or a ResizeConfiguration object
      * @param string|ResizeOptions|null          $options The target path as string or a ResizeOptions object
-     *
-     * @return ImageInterface
      */
-    public function create($path, $size = null, $options = null)/*: ImageInterface*/;
+    public function create(string|ImageInterface $path, int|array|ResizeConfiguration|null $size = null, string|ResizeOptions|null $options = null): ImageInterface;
```

Comment:
```suggestion
    public function create(string|ImageInterface $path, int|string|array|ResizeConfiguration|null $size = null, string|ResizeOptions|null $options = null): ImageInterface;
```

See https://github.com/contao/contao/blob/602bd1fe3db1211ec708749d5564207f73e9f3d1/core-bundle/src/Image/ImageFactory.php#L71 and https://github.com/contao/contao/blob/602bd1fe3db1211ec708749d5564207f73e9f3d1/core-bundle/src/Image/ImageFactory.php#L113